### PR TITLE
Refactor `Command`

### LINKED ISSUE #155: Refactor `Command` classes
Reduce code repetition, streamline `LogicManager` code.

### PR DESCRIPTION
**Pull request by seanlim**
Closes #155

### CODE REVIEW

File: src/test/java/spleetwaise/address/logic/parser/CommandParserTestUtil.java [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static void assertParseSuccess(
             Command expectedCommand
     ) {
         try {
-            Command command = parser.parse(userInput);
+            spleetwaise.commons.logic.commands.Command command = parser.parse(userInput);
```
Comment (SeeYangZhi) [inline src/test/java/spleetwaise/address/logic/parser/CommandParserTestUtil.java:22]: Is it necessary to have the explicit declaration here for the `commons.logic.Command`? 
Above have imported `spleetwaise.commons.logic.commands.Command;` at line 6

**Review by SeeYangZhi — APPROVED** [review]
LGTM, logic manager is more intuitive to understand now.

File: src/test/java/spleetwaise/address/logic/parser/CommandParserTestUtil.java (hunk 1 above)
Comment (seanlim) [reply src/test/java/spleetwaise/address/logic/parser/CommandParserTestUtil.java:22]: Good catch, will rectify and merge

Comment (SeeYangZhi) [reply src/test/java/spleetwaise/address/logic/parser/CommandParserTestUtil.java:22]: 谢谢

**Pull request merged**